pkg/agent/route/route_windows.go

```diff
@@ -165,7 +191,7 @@ func (c *Client) listRoutes() (map[string]*netroute.Route, error) {
 	rtMap := make(map[string]*netroute.Route)
 	for idx := range routes {
 		rt := routes[idx]
-		if rt.LinkIndex != c.nodeConfig.GatewayConfig.LinkIndex {
+		if rt.LinkIndex != c.nodeConfig.GatewayConfig.LinkIndex && rt.LinkIndex != c.bridgeInfIndex {
```

Comment:
You need to remove the second condition too? otherwise some routes on br-int will be removed unexpectedly.  We only manage routes on antrea-gw0 so should return them only.

Reply:
Updated.